jdaviz/configs/specviz/plugins/unit_conversion/unit_conversion.py

```diff
@@ -237,9 +236,13 @@ def _on_add_data_to_viewer(self, msg):
 
                     try:
                         if angle_unit is None:
-                            # default to sr if input spectrum is not in surface brightness units
-                            # TODO: for cubeviz, should we check the cube itself?
-                            self.angle_unit.selected = 'sr'
+                            if self.config in ['specviz', 'specviz2d']:
+                                self.has_angle = False
+                                self.has_sb = False
```

Comment:
We don't append an angle unit in Specviz right now, see the screenshot when the native unit is Jy. This hides both from displaying in the UI and unexposes them from the API. 

<img width="219" alt="Screenshot 2024-12-19 at 4 40 57 PM" src="https://github.com/user-attachments/assets/4adba1a9-ef8e-47b1-87c6-3c3bae7c7bd4" />